src/generic/clipboard/paste-component/index.jsx

```diff
@@ -30,10 +30,10 @@ const PasteComponent = ({
       </Popover>
     </div>
   );
-
+  const textt = (clipboardData.sourceContextTitle || "").startsWith("lib:") ? "Link library component from clipboard" : text;
```

Comment:
You can use [`isLibraryKey`](https://github.com/openedx/frontend-app-authoring/blob/2cd77ce4559d4577c875a959cb32c741c3facabd/src/generic/key-utils.ts#L33C17-L33C29)

```suggestion
  const textt = isLibraryKey(clipboardData.sourceContextTitle) ? "Link library component from clipboard" /* TODO: needs i18n */ : text;
```

Reply:
👍🏻